larray/inout/excel.py

```diff
@@ -139,6 +139,13 @@ def __init__(self, filepath=None, overwrite_file=False, visible=None, silent=Non
             if silent is None:
                 silent = not visible
 
+            # activate XLA(M) addins
```

Comment:
I think we should only do this when creating the App instance, not for "active" nor explicit app= given by user (and to a lesser extent, not when reusing an **existing** global app).